plugins/CoreHome/angularjs/common/services/periods.js

```diff
@@ -63,7 +63,7 @@
         },
 
         getDateRange: function () {
-            return [this.dateInPeriod, this.dateInPeriod];
+            return [new Date(this.dateInPeriod.getTime()), new Date(this.dateInPeriod.getTime())];
```

Comment:
we need to ensure new date objects are returned here. Otherwise only references for `this.dateInPeriod` were returned, causing problems when they were manipulated. (That caused day periods compared two years back)